codegen-core/common-test-models/naming-obstacle-course-casing.smithy

```diff
@@ -0,0 +1,63 @@
+$version: "1.0"
+namespace casing
+
+use aws.protocols#awsJson1_1
+
+// TODO(https://github.com/awslabs/smithy-rs/issues/2340): The commented part of the model breaks the code in a miriad
+// of ways. The solution to the linked issue must solve them.
+
+/// Confounds model generation machinery with lots of problematic casing
+@awsJson1_1
+service ACRONYMInside_Service {
+    operations: [
+        DoNothing,
```

Comment:
We don't need this operation if https://github.com/awslabs/smithy-rs/pull/2351 is merged